### ISSUE TITLE
Service Idea: Proactive Taxes

### ISSUE THREAD

**Issue by nichbarry** [usds/playbook#123]
Instead of requiring someone to file their taxes, the government should file with the person. Every year the government should say: based on what you earned, we think you owe/we owe [this much]. From there, there would be a massive report on all the spending / earning data a person has for the tax year (which you have on file [and if you don't team up with all of those banks, or just convert every bank to FCU, or team up with Bitcoin {obviously, you would get permission from individuals and businesses}]). Increased transparency. Then, the person can counter and say actually I think I owe this much, accept their refund as is or waffle. Is not a primary point of taxes to keep tabs on people? So be proactive in your tabbing; go out and get it government.

A person would be able to incrementally input information on their online tax hub throughout the year (or it could be cached locally for maximum privacy). Always offer a private, locally stored option for all services with the ability to incrementally upload. Could be extended to businesses (ie, incremental upload needed quarterly)

In summary: aggregate and transparensize reporting practices
